protobufs/livekit_sip.proto

```diff
@@ -671,3 +671,12 @@ enum SIPCallDirection {
   SCD_INBOUND = 1;
   SCD_OUTBOUND = 2;
 }
+
+message SIPCall {
```

Comment:
How does this compare to the existing SIPCallInfo type? Can we also try to keep the fields names consistent between the different types? For instance, SIPCallInfo has a `call_id` field that, I believe` contains the same value.